testsuite/texture-udim/run.py

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env python
 
-
-command += oiiotool ("-pattern constant:color=.5,.1,.1 256x256 3 -text:size=50:x=75:y=140 1001 -d uint8 -otex file.1001.tx")
-command += oiiotool ("-pattern constant:color=.1,.5,.1 256x256 3 -text:size=50:x=75:y=140 1002 -d uint8 -otex file.1002.tx")
-command += oiiotool ("-pattern constant:color=.1,.1,.5 256x256 3 -text:size=50:x=75:y=140 1011 -d uint8 -otex file.1011.tx")
-command += oiiotool ("-pattern constant:color=.1,.5,.5 256x256 3 -text:size=50:x=75:y=140 1012 -d uint8 -otex file.1012.tx")
+command += oiiotool ("-pattern constant:color=.5,.1,.1 256x256 3 -text:size=50:x=75:y=140 1001 --attrib Make pet --attrib Model dog -d uint8 -otex file.1001.tx")
```

Comment:
It'd be good to add tests for all the patterns.

Reply:
Yeah, that's a good point, thanks for holding my feet to the fire on this.